### PR TITLE
Release Notes 113 for WebView2

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article for review:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2549
[[gh rendered](https://github.com/mikehoffms/edge-developer/blob/user/mikehoffms/relnotes-113/microsoft-edge/webview2/release-notes.md)]

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:1]: fix bad tagging near https://learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?tabs=dotnetcsharp#bug-fixes-5
symptom: excess clickable blue text

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
...
 The following sections cover either a Release SDK package (1.0.####.##) or a Prerelease SDK package (1.0.####-prerelease).
 
+<!-- ====================================================================== -->
+## 1.0.1774.30
+
+Release Date: May 8, 2023
+
+[NuGet package for WebView2 SDK 1.0.1774.30](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 112.0.1774.30 or higher.
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:114]:
```suggestion
For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 113.0.1774.30 or higher.
```
@nishitha-burman @novac42 - change from 112 to 113?

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
...
+
+[NuGet package for WebView2 SDK 1.0.1829-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1829-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 115.0.1829.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### General
+
+
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:193]: Add Promotions section?  

```
###### Promotions

The following APIs have been promoted to Stable and are now included in this Release SDK.
```

not seeing strict alternation in Search POV, so not sure if the Promotions section is always present:
image

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
...
+<!-- ------------------------------ -->
+###### Experimental features
+
+* The `IsPasswordAutosaveEnabled` API to
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:197]: todo, give description eg for student devs

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
...
+   * [CoreWebView2Profile.CookieManager property](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.cookiemanager?view=webview2-dotnet-1.0.1774.30&preserve-view=true#microsoft-web-webview2-core-corewebview2profile-cookiemanager)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2Profile`
+   * [CoreWebView2Profile.CookieManager property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1774.30&preserve-view=true#cookiemanager)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Profile5](/microsoft-edge/webview2/reference/win32/icorewebview2profile5?view=webview2-1.0.1774.30&preserve-view=true)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:174]:
```suggestion
* `ICoreWebView2Profile5`
```
per latest conventions, can unlink, like .net.  There's no info section at top of narrow type needing link:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2profile5?view=webview2-1.0.1777-prerelease&preserve-view=true&viewFallbackFrom=webview2-1.0.1774.30

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -94,6 +105,141 @@ SDKs/packages are Prerelease or Release
...
+
+
+<!-- ------------------------------ -->
+#### General
+
+
+<!-- ------------------------------ -->
+###### Experimental features
+
+* The `IsPasswordAutosaveEnabled` API to tbd<!--todo-->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:204]: @novac42 @nishitha-burman - need description understandable by a dev student

File: microsoft-edge/webview2/release-notes.md
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:1]: fixed.  test them...

File: microsoft-edge/webview2/release-notes.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: unlinked

File: microsoft-edge/webview2/release-notes.md
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:1]: passed

File: microsoft-edge/webview2/release-notes.md (hunk 1 above)
Comment (peiche-jessica) [reply microsoft-edge/webview2/release-notes.md]: 113 is correct.

File: microsoft-edge/webview2/release-notes.md [hunk 6]
```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
...
+   * [ICoreWebView2Profile6::put_IsGeneralAutofillEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#put_isgeneralautofillenabled)
+---
+
+<!-- ------------------------------ -->
+###### Bug fixes
+
+* Disable the Chrome Web Store info banner that displays the option to allow extensions installation. ([Issue #3312](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3312))
+* [TODO: Add description here] ([Issue #3300](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3300))
+* [TODO: Add description here] ([Issue #3375](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3375))
+* Fixed a bug in PostSharedBufferToScript ([Issue #3360] (https://github.com/MicrosoftEdge/WebView2Feedback/issues/3360)). Runtime only.
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/release-notes.md]:
```suggestion
* Fixed a bug in PostSharedBufferToScript ([Issue #3360](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3360)). Runtime only.
```

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: checking latest Files Changed tab...
present view/listing is Outdated.

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: the problem remains.  fixing...

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:221]: Question: Are there Promotions sections that are empty, in previous sections?  
Answer: No; per a recent normalization pass, any Promotions sections that exist have content and lead-in, either:
The following APIs have been promoted to Stable and are now included in this Release SDK.
The following APIs have been promoted from Experimental to Stable in this Prerelease SDK.

There are 57 #### General   headings/sections
There are 39 ###### Promotions   headings/sections

The policy/design is evidently:  omit Promotions if there are no promotions; don't do:
```
###### Promotions
There are no Promotions in this release|prerelease.
```

File: microsoft-edge/webview2/release-notes.md (hunk 5 above)
Comment (novac42) [reply microsoft-edge/webview2/release-notes.md]: Added

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (novac42) [reply microsoft-edge/webview2/release-notes.md:219]: Seems we don't have experimental features, only pre-release stable this time. I have adjusted the heading accordingly.

File: microsoft-edge/webview2/release-notes.md [hunk 7]
```diff
@@ -94,6 +105,179 @@ SDKs/packages are Prerelease or Release
...
+   * [CoreWebView2Profile.IsGeneralAutofillEnabled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile#isgeneralautofillenabled?view=webview2-winrt-1.0.1829-prerelease&preserve-view=true)
+   * [CoreWebView2Profile.IsPasswordAutosaveEnabled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile#ispasswordautosaveenabled?view=webview2-winrt-1.0.1829-prerelease&preserve-view=true)
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+* `ICoreWebView2Profile6` interface:
+   * [ICoreWebView2Profile6::get_IsPasswordAutosaveEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#get_ispasswordautosaveenabled)
+   * [ICoreWebView2Profile6::put_IsPasswordAutosaveEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#put_ispasswordautosaveenabled)
+   * [ICoreWebView2Profile6::put_IsPasswordAutosaveEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#put_ispasswordautosaveenabled)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:244]:
```suggestion
```
dup

File: microsoft-edge/webview2/release-notes.md [hunk 8]
```diff
@@ -94,6 +105,179 @@ SDKs/packages are Prerelease or Release
...
+
+* `CoreWebView2Profile` Class:
+   * [CoreWebView2Profile.IsGeneralAutofillEnabled Property](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.isgeneralautofillenabled?view=webview2-dotnet-1.0.1829-prerelease&preserve-view=true)
+   * [CoreWebView2Profile.IsPasswordAutosaveEnabled Property](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.ispasswordautosaveenabled?view=webview2-dotnet-1.0.1829-prerelease&preserve-view=true)
+
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2Profile` Class:
+   * [CoreWebView2Profile.IsGeneralAutofillEnabled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile#isgeneralautofillenabled?view=webview2-winrt-1.0.1829-prerelease&preserve-view=true)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:235]:
```suggestion
   * [CoreWebView2Profile.IsGeneralAutofillEnabled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1829-prerelease&preserve-view=true#isgeneralautofillenabled)
```

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:215]: @novac42 @peiche-jessica 
i removed hidden comment
```
<!-- Seems we don't have experimental features, only pre-release stable this time.-->
```
and added standard string:
```
The following APIs have been promoted from Experimental to Stable in this Prerelease SDK.
```

File: microsoft-edge/webview2/release-notes.md [hunk 9]
```diff
@@ -93,6 +94,183 @@ SDKs/packages are Prerelease or Release
...
+   * [ICoreWebView2Profile6::get_IsGeneralAutofillEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#get_isgeneralautofillenabled)
+   * [ICoreWebView2Profile6::put_IsGeneralAutofillEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#put_isgeneralautofillenabled)
+
+---
+
+
+<!-- ------------------------------ -->
+###### Bug fixes
+
+* Fixed an issue to allow an app to inject initial scripts by calling `AddScriptToExecuteOnDocumentCreated` before a new window is created.  ([Issue #2491](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2491))
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:228]: @novac42 @peiche-jessica -- Should this item and the next (X-Edge-Shopping-Flag) be in this Prerelease > BugFixes section, or in a Release > BugFixes section like in [this Commit](https://github.com/MicrosoftDocs/edge-developer/pull/2549/commits/7b738d801708c2821a1873441702c089080735e5)?

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: i restored the Release > Bug fixes section.  
some bug fixes were cherry picked to stable release

**Review by novac42 — APPROVED** [review]

File: microsoft-edge/webview2/release-notes.md [hunk 10]
```diff
@@ -84,7 +74,18 @@ Async methods:
 <!-- ------------------------------ -->
 #### Experimental APIs, Prerelease SDKs, and Release SDKs
 
-APIs are initially introduced as Experimental APIs.  Then they become Stable APIs in a Prerelease SDK package, and soon after, they become Stable APIs in a Release SDK package.  For more information, see [Phases of introducing APIs](./concepts/versioning.md) in _Understand the different WebView2 SDK versions_.
+New APIs are added in phases, as follows:
+
+1. APIs are initially introduced as Experimental APIs in a Prerelease SDK package.
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:79]: if a line wraps, i generally space further apart.  might tighten.

**Review by peiche-jessica — APPROVED** [review]

**Pull request merged**